beacon_node/eth2_libp2p/src/peer_manager/peer_info.rs

```diff
@@ -46,6 +47,9 @@ pub struct PeerInfo<T: EthSpec> {
     /// Direction of the first connection of the last (or current) connected session with this peer.
     /// None if this peer was never connected.
     pub connection_direction: Option<ConnectionDirection>,
+    /// The enr of the peer, if known.
+    #[serde(skip)]
```

Comment:
do we need to skip the serialization? 
Enr has a `serde` feature which implements this. Might be nice to have.

Reply:
removed the skip 👍